package.json

```diff
@@ -37,9 +37,15 @@
     "@18f/identity-normalize-yaml": "^1.0.0",
     "@axe-core/puppeteer": "^4.5.1",
     "@babel/eslint-plugin": "^7.14.5",
+    "@testing-library/dom": "^9.3.1",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
     "@types/jest": "^29.2.6",
     "@typescript-eslint/eslint-plugin": "^5.48.2",
     "@typescript-eslint/parser": "^5.48.2",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
Are these necessary? Especially the stage 2 preset. Where is that used?

Reply:
I was trying to work around some module import issues. I found a working solution, so these extra deps are now removed.